tutorials/tutorial-hello-timescale.md

```diff
@@ -0,0 +1,803 @@
+dropoff_datetime      | 2016-01-01 00:11:14
+passenger_count       | 1
+trip_distance         | 6.00
+pickup_longitude      | -73.947151184082031
+pickup_latitude       | 40.791046142578125
+rate_code             | 1
+dropoff_longitude     | -73.920768737792969
+dropoff_latitude      | 40.865577697753906
+payment_type          | 2
+fare_amount           | 18
+extra                 | 0.5
+mta_tax               | 0.5
+tip_amount            | 0
+tolls_amount          | 0
+improvement_surcharge | 0.3
+total_amount          | 19.3
+```
+
+### Mission 2: Analysis
+
+Let's say that the NYC Taxi and Limousine Commission has made it a key
+goal to mitigate the impact of global warming by reducing their greenhouse 
+gas emissions by 20% by 2024. Given the number of taxi rides taken each 
+day, they believe studying past taxi rider history and behavior will enable 
+them to plan for the future.
+
+In this tutorial, we will limit analysis of historical taxi ride data
+to all NYC TLC taxi rides taken in January 2016. You can imagine that in a
+more expansive scenario, you will want to examine rides taken over several years.
+
+#### How many rides took place on each day?
+
+The first question you will explore is simple: *How many rides took place on each day during January 2016?*
+
+Since TimescaleDB supports full SQL, all that’s required is a simple SQL query 
+to count the number of rides and group/ order them by the day they took place, 
+as seen below:
+
+```sql
+-- What's the total number of rides that took place everyday for first 5 days
+SELECT date_trunc('day', pickup_datetime) as day, COUNT(*) FROM rides GROUP BY day ORDER BY day;
+```
+
+With this information, we know *how many* rides are taken each day, and we can
+identify the days of the week and month in which the most rides take place.
+Your result should look like this:
+
+```sql
+         day         | count  
+---------------------+--------
+ 2016-01-01 00:00:00 | 345037
+ 2016-01-02 00:00:00 | 312831
+ 2016-01-03 00:00:00 | 302878
+ 2016-01-04 00:00:00 | 316171
+ 2016-01-05 00:00:00 | 343251
+ 2016-01-06 00:00:00 | 348516
+ 2016-01-07 00:00:00 | 364894
+ 2016-01-08 00:00:00 | 392070
+ 2016-01-09 00:00:00 | 405825
+ 2016-01-10 00:00:00 | 351788
+ 2016-01-11 00:00:00 | 342651
+ 2016-01-12 00:00:00 | 367390
+ 2016-01-13 00:00:00 | 395090
+ 2016-01-14 00:00:00 | 396473
+ 2016-01-15 00:00:00 | 401289
+ 2016-01-16 00:00:00 | 411899
+ 2016-01-17 00:00:00 | 379156
+ 2016-01-18 00:00:00 | 341481
+ 2016-01-19 00:00:00 | 385187
+ 2016-01-20 00:00:00 | 382105
+ 2016-01-21 00:00:00 | 399654
+ 2016-01-22 00:00:00 | 420162
+ 2016-01-23 00:00:00 |  78133
+ 2016-01-24 00:00:00 | 159766
+ 2016-01-25 00:00:00 | 282087
+ 2016-01-26 00:00:00 | 327655
+ 2016-01-27 00:00:00 | 359180
+ 2016-01-28 00:00:00 | 383326
+ 2016-01-29 00:00:00 | 414039
+ 2016-01-30 00:00:00 | 435369
+ 2016-01-31 00:00:00 | 361505
+ 2017-11-17 00:00:00 |      2
+(32 rows)
+```
+
+#### What is the average fare amount for 2+ passengers?
```

Comment:
If you are switching everything over to one passenger, this header needs to change as well